### ISSUE TITLE
Error: Key not present in Dictionary: getVersion(), [0, 1]) when using eslint

### ISSUE THREAD

**Issue by jedverity** [scrooloose/syntastic#1426]
I added `let g:syntastic_javascript_checkers = ['eslint']` to my vimrc and got the following error in the writeposthook:

```
Error detected while processing function <SNR>85_BufWritePostHook..<SNR>85_UpdateErrors..<SNR>85_CacheErrors..269..SyntaxCheckers_javascript_eslint_IsAvailable:
line    4:
E716: Key not present in Dictionary: getVersion(), [0, 1])
E116: Invalid arguments for function syntastic#util#versionIsAtLeast(self.getVersion(), [0, 1])
E15: Invalid expression: syntastic#util#versionIsAtLeast(self.getVersion(), [0, 1])
```

I made sure everything (syntastic, eslint) was up to date with the latest - still no dice. Anyone have thoughts?

Vim 7.4.622

**Comment by jedverity** [comment]
:SyntasticInfo output:

```
Syntastic version: 3.5.0-72
Info for filetype: javascript.jsx
Global mode: active
Filetype javascript.jsx is active
The current file will be checked automatically
Available checkers: -
Currently enabled checkers: -
```

So eslint obviously not active. I've confirmed that it's available on my PATH and works when run on files manually.

I also tried specifying the path manually with `let g:syntastic_javascript_eslint_exec` but still no dice. Must be missing something simple.

Also tried swapping in jsx for javascript in settings, all to no avail.
